### PR TITLE
Updated terminology, the task assign response and the volume definition

### PR DESCRIPTION
**Pull request by illybwaa**
The task assign example response was quite different from what the server was actually returning, so I changed that to reflect what the server responds with at the current time. Some of the values in the response aren't exactly known to me so those still have to be filled in.

The terminology had some incorrect parts in it and others which needed more description so I changed what I saw was wrong.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,17 +38,17 @@ The EyeWire REST API provides programmatic access to interact with EyeWire's dat
...
+    - a set of 8bpp greyscale .jpg files
     - the electron microscope images
 - `segmentation`
-    - a .png file
+    - a set of 24bpp RGB .png files for normal `tasks` or 32bpp RGBA .png files for omni `tasks`
     - used to describe which pixels belong to which `segment` ID
-    - The RGBA color represents the `segment` ID: R + 256*G + 256^2 * B + 256^3 * A
-  `voxel`
-      the highest precision of the dataset. Can be considered a 1x1x1 pixel cube.
+    - The RGBA color represents the `segment` ID: R + 256*G + 256^2 * B + 256^3 * A. The alpha channel is excluded in non-omni `tasks`.
```
Comment (chrisj) [inline README.md]: I think we should remove references to omni or non omni. For now, the api is focusing on non-omni (eyewire). I would change the equation to just `R + 256*G + 256^2 * B` which is the equation I use in the sample app.

**Comment by illybwaa** [comment]
Changed it so it doesn't reference omni-only things and also changed call name from save to submit

File: README.md [hunk 2]
```diff
@@ -212,43 +216,86 @@ https://beta.eyewire.org/2.0/tasks/assign?access_token=78q3ja8y
 
 ```json
 {
-  "id": 17444,
-  "seeds": [245, 5025, 6500],
-  "cell": 10,
+  "id": 558593,
```
Comment (chrisj) [inline README.md]: this change shouldn't be needed anymore. api is updated to fit the documentation.

**Comment by illybwaa** [comment]
Should be fine now

**Pull request merged**